patchwork/models.py

```diff
@@ -245,6 +245,9 @@ class State(models.Model):
     slug = models.SlugField(max_length=100, unique=True)
     ordering = models.IntegerField(unique=True)
     action_required = models.BooleanField(default=True)
+    review_intention_expiration_time = models.DurationField(
+        default=datetime.timedelta(days=30)
+    )
```

Comment:
I'll presumably figure this out later, but why are these attributes of `State` rather than e.g. `Project`? What has `State` got to do with things?

Also, as a larger point: do we actually need to do this in Patchwork itself? Once we have a date associated with the attention set entry, we could leave removal to an external tool via the REST API, right? Is there a good reason not to do that?